### PR TITLE
Blog - Hawtio v3 release and Camel CLI

### PR DESCRIPTION
**Pull request by tadayosi**
Hi, my first post to Camel Blog. Please review this. Thanks!

@davsclaus @oscerd @apupier @lhein

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/blog/2024/01/hawtio-v3/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+**New plugin system:** \
+Hawtio has always been easy to extend by developing your own plugins, but in v3 the plugin system has been redesigned to comply with a new standard for micro frontends, [Webpack Module Federation](https://webpack.js.org/concepts/module-federation/). At the same time, a number of sample projects are available for interested users to easily try out the plugin development.
+
+**New CLI:** \
+Since the introduction of [JBang](https://www.jbang.dev/), the provision of a JBang-based CLI has become the new standard method for distribution in the Java ecosystem. Hawtio has been able to be launched from the Camel CLI (a.k.a. [Camel JBang](https://camel.apache.org/manual/camel-jbang.html)), but starting with Hawtio v3, the JBang-based Hawtio CLI is introduced as the new distribution method replacing the previous `hawtio-app.jar`. It allows you to install and launch the latest Hawtio from the JBang catalog with a single command if you have JBang installed locally.
```
Comment (apupier) [inline content/blog/2024/01/hawtio-v3/index.md:34]: internal link must be relative (this should fail the build)

Comment (tadayosi) [reply content/blog/2024/01/hawtio-v3/index.md:34]: Good catch. Will fix.

**Review by squakez — APPROVED** [review]

**Comment by zregvart** [comment]
The preview should be fixed now, but due to the context of this PR it will not run with the fixed version. I don't see the need to rebase and re-run checks to get the preview. So feel free to merge.

**Comment by tadayosi** [comment]
Thanks, so I'll just change the publish date and merge it.

**Pull request merged**